inc/Engine/Common/JobManager/JobProcessor.php

```diff
@@ -582,4 +582,26 @@ private function make_status_pending( string $url, string $job_id, string $queue
 			$factory->manager()->make_status_pending( $url, $job_id, $queue_name, $is_mobile, $optimization_type );
 		}
 	}
+
+	/**
+	 * Send the link to Above the fold SaaS.
+	 *
+	 * @param string $url Url to be sent.
+	 * @return array
+	 */
+	public function add_to_atf_queue( string $url ): array {
```

Comment:
Good point, the why behind I used it here is to use the is_home method that is already defined here and I didn't want to touch any other part of code in case I wanted to move is_home from here but let me check and get another better idea.

Reply:
Done